test/api/context/ContextDefinition.ts

```diff
@@ -45,7 +46,9 @@ export default class ContextDefinition {
   static readonly SITE_AREA_CONTEXTS: any = {
     NO_SITE: 'No site', // Used for unassigned Charging Station or CS in tenant with no organizations
     WITH_ACL: 'withACL', // ACL is active
-    WITHOUT_ACL: 'withoutACL' // ACL is inactive
+    WITHOUT_ACL: 'withoutACL', // ACL is inactive
+    WITH_SMART_CHARGING_THREE_PHASED: 'withSmartChargingThreePhased', // Smart Charging is active three phased
```

Comment:
Just for info, we will have also in the future WITH_SMART_CHARGING_DC (3 phases always)